src/xrGame/ui/UIActorMenu.cpp

```diff
@@ -975,3 +975,63 @@ bool CUIActorMenu::CanSetItemToList(PIItem item, CUIDragDropListEx* l, u16& ret_
 
     return false;
 }
+
+void CUIActorMenu::HighlightSectionInSlot(pcstr section, EDDListType type, u16 slot_id /*= 0*/)
+{
+    CUIDragDropListEx* slot_list = GetListByType(type);
+
+    if (!slot_list)
+        slot_list = m_pLists[eInventoryBagList];
+
+    u32 const cnt = slot_list->ItemsCount();
+    for (u32 i = 0; i < cnt; ++i)
+    {
+        CUICellItem* ci = slot_list->GetItemIdx(i);
+        const PIItem item = static_cast<PIItem>(ci->m_pData);
+        if (!item)
+            continue;
+
+        if (!strcmp(section, item->m_section_id.c_str()) == 0)
```

Comment:
Strange condition: both `! ` and `== 0` are being used.
Please, use `strcmp != 0` or `strcmp == 0`.

Reply:
Fixed